pkg/virt-operator/install-strategy/create.go

```diff
@@ -1392,13 +1394,19 @@ func addOrRemoveSSC(targetStrategy *InstallStrategy,
 		}
 
 		if modified {
+			oldUserBytes, err := json.Marshal(oldUsers)
+			if err != nil {
+				return err
+			}
 			userBytes, err := json.Marshal(users)
 			if err != nil {
 				return err
 			}
 
-			data := []byte(fmt.Sprintf(`{"users": %s}`, userBytes))
-			_, err = scc.SecurityContextConstraints().Patch(sccPriv.TargetSCC, types.StrategicMergePatchType, data)
+			test := fmt.Sprintf(`{ "op": "test", "path": "/users", "value": %s }`, string(oldUserBytes))
+			patch := fmt.Sprintf(`{ "op": "replace", "path": "/users", "value": %s }`, string(userBytes))
```

Comment:
Is it possible to ever have an empty or omitted `/users` struct? I've found that `replace` can error unexpectedly if e.g. `/users` was missing entirely. in which case `add` needs to be used instead (and no test is needed as there's an implicit "and /users must not already exist")

Reply:
Confirmed to be a non-issue

Reply:
Good point. It looks like we're okay in this instance though.

I tested this by creating an SCC rule that omitted the 'users' array in the yaml file. Then I patched that SCC object with the "replace" op. It worked.